spotbugs/src/test/java/edu/umd/cs/findbugs/filter/SourceMatcherTest.java

```diff
@@ -118,6 +122,45 @@ public void match() throws Exception {
         assertFalse(sm.match(bug));
     }
 
+    @Test
+    public void testRealPathMatchRegexp() throws Exception {
```

Comment:
A more descriptive name would be good. What behavior do we expect?

Also, this test is very long and should probably be split.

Reply:
Well, tried to split the test case. Not super happy with it. Any suggestions for it?